apps/quick-learn-frontend-e2e/src/cypress/e2e/Profile.cy.ts

```diff
@@ -0,0 +1,43 @@
+import LoginPage = require("../test/Login");
+import profile = require("../test/Profile");
+
+describe('Primary Skill Update', () => {
+    const loginPage = new LoginPage();
+
+  beforeEach(() => {
+    loginPage.visit();
+    cy.get('.text-xl').contains("Sign in to your account")
```

Comment:
same

Reply:
It will take time I will mention where will you need to add ID